content/en/docs/home/_index.md

```diff
@@ -60,7 +60,7 @@ cards:
   title: K8s Release Notes
   description: If you are installing Kubernetes or upgrading to the newest version, refer to the current release notes.
   button: "Download Kubernetes"
-  button_path: "/docs/setup/release/notes"
+  button_path: "/releases/download"
```

Comment:
LGTM 💯 , thanks for the patch @PriyanshuAhlawat, I tested its going to correct link now (not a official approval)